src/main/kotlin/com/sourcegraph/cody/initialization/UninstallListener.kt

```diff
@@ -12,13 +12,18 @@ import com.sourcegraph.cody.agent.protocol.BillingProduct
 import com.sourcegraph.cody.agent.protocol.TelemetryEventParameters
 import com.sourcegraph.cody.config.CodyAuthenticationManager
 import com.sourcegraph.cody.telemetry.TelemetryV2
+import com.sourcegraph.config.ConfigUtil
 import java.util.concurrent.TimeUnit
 
 class UninstallListener : StartupActivity {
```

Comment:
I noticed it looks like it is currently broken:

![image](https://github.com/user-attachments/assets/56bfd113-5292-4fd3-95fd-851fc90267e8)